pkg/config/config.go

```diff
@@ -730,3 +731,31 @@ func (lci *LocalConfigInfo) GetOSSourcePath() (path string, err error) {
 
 	return sourceOSPath, err
 }
+
+// IsValidArgumentList validates the argument list for `odo config` command
+func IsValidArgumentList(args []string) error {
+	if len(args) == 0 {
+		return nil
+	}
+
+	param, ok := asLocallySupportedParameter(args[0])
+
+	if !ok {
+		return errors.Errorf("the parameter provided : %v, is not present in the supported parameters list.", args[0])
```

Comment:
There is an error string convention that error don’t start with capitals or end with a ### fullstop